canonical-websites/snapcraft.io
# Image restrictions

Fixes https://github.com/canonical-websites/snapcraft.io/issues/1674
Fixes https://github.com/canonical-websites/snapcraft.io/issues/413

## Done
- Only trigger a form state update when the state changes in a meaningful way
- ~Added a new simplified "p-overlay" pattern (to be rolled out to other pages (snap details))~
- Added the restrictions text

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/<snap_name>/listing
- See the media restrictions overlay when you click the button
- Ensure media uploading/ removing and saving still works
